apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -11,42 +11,35 @@ import {
   theme,
 } from '@aragon/ui'
 import { VOTE_ABSENT } from '../vote-types'
-import { isVoteOpen, getAccountVote } from '../vote-utils'
 import VoteSummary from './VoteSummary'
 import VoteStatus from './VoteStatus'
 
-const VotePanelContent = ({
-  vote: { id, vote, endDate, creatorName, metas: { question } },
-  user,
-  tokenSupply,
-  support,
-  voteTime,
-  ready,
-}) => {
+const VotePanelContent = ({ vote, user, ready }) => {
   if (!vote) {
     return null
   }
 
-  const { minAcceptQuorumPct: quorum, creator, yea, nay, voters } = vote
-  const userVote = getAccountVote(user.address, voters)
-  const opened = isVoteOpen(vote, voteTime)
+  const { question, quorum, support, endDate } = vote
+  const { creator, yea, nay, totalVoters } = vote.vote
+
+  const creatorName = 'Robert Johnson' // TODO: get creator name
+  const accountVote = VOTE_ABSENT // TODO: detect if the current account voted
```

Comment:
This should make this a lot easier :)

https://github.com/aragon/aragon-apps/pull/103

Reply:
Totally! Thanks 💖